### ISSUE TITLE
Missing DRC rules

### ISSUE THREAD

**Issue by mguthaus** [efabless/sky130_klayout_pdk#30]
## Expected Behavior

It should implement all checks for FEOL and BEOL. 

## Actual Behavior

It does not check a lot of things:
poly.7 (extension of poly beyond diff)
licon.5a (diff enclose licon)

## Steps to Reproduce the Problem

Run DRC on attached layout.
[sky130_fd_sc_hd__inv_1-errors.zip](https://github.com/user-attachments/files/18171683/sky130_fd_sc_hd__inv_1-errors.zip)

## Specifications

- Version:
- Platform:

**Comment by DavidRLindley** [comment]
@mguthaus You are correct and we are working on it.  I plan to roll out updates at the end of February.

**Comment by mguthaus** [comment]
This PDK had many of the FEOL rules completed:

https://github.com/laurentc2/SKY130_for_KLayout

**Comment by mguthaus** [comment]
I integrated them with a PDK similar to yours here:

https://github.com/VLSIDA/chip-tutorials/blob/main/klayout/tech/sky130/drc/sky130A.drc
